src/patterns/atoms/Footer/Footer.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../theme/common.scss";
```

Comment:
I'd recommend to use absolute imports to design system. Let's discuss

Reply:
Wooooooow this is just amazing
includePaths:[path.resolve(__dirname, 'src/patterns/')]

solved thanks

Reply:
implemented <3